pkg/formatter/table.go

```diff
@@ -72,11 +72,13 @@ func WriteNodeReport(records []*reporting.NodeReportItem) {
 	lines := strings.SplitN(bufStr, "\n", 2)
 	width := tablewriter.DisplayWidth(lines[0])
 
+	var warning strings.Builder
 	fmt.Printf(bufStr)
 	if termWidth < width {
-		fmt.Print("\nNote:  To view the report with correct formatting, please expand")
-		fmt.Printf("\n       your terminal window to be at least %v characters wide\n", width)
+		warning.WriteString(fmt.Sprintf("\nNote:  To view the report with correct formatting, please expand"))
+		warning.WriteString(fmt.Sprintf("\n       your terminal window to be at least %v characters wide\n", width))
 	}
+	return buffer.String(), warning.String()
```

Comment:
The variable here returned is called `warning`, but above you call it `errStr`. which one is it?
```
			reportStr, errorStr := formatter.FormatNodeReport(results)
```